### ISSUE TITLE
swaggerhub Responses examples JSON responses that used to show proper formatting no longer do

### ISSUE THREAD

**Issue by frankmainframecloud** [swagger-api/swagger-ui#6743]
### Browsers tested
- Google Chrome Version 87.0.4280.88 (Official Build) (64-bit)
- Microsoft Edge Version 87.0.664.60 (Official build) (64-bit)

### Problem Description
To see the problem

1.  go to this URL [https://app.swaggerhub.com/apis-docs/zData9MfC/mainframe-cloud_file_service/0.2#/read/post_read]

2. scroll down to Responses section

3. look at Example Value

4. what you see now is a string with no formatting (as shown below for one of the examples).

`"{\n    \"rc\": 200,\n    \"agentVersion\": \"ON-DEMAND\",\n    \"respobj\": \"cmdresp\",\n    \"cmdresp\": {\n        \"data\": [\n            \"10010TOM  USR400\",\n            \"10020MARK USR300\",\n            \"10030MARK USR200\",\n            \"10040BILL USR100\",\n            \"10060HARRYUSR405\",\n            \"10070JILL USR404\"\n        ],\n        \"file\": \"'MSP.VSAM.DATA(REPRO1)'\",\n        \"rc\": 0\n    }\n}"`

What you SHOULD see, and what you USED to see was a well formatted JSON object (same formatting as the Request body examples which are still good).

We have not changed our swagger definitions in months, so it appears a change in swaggerhub must have caused this.

### WORK AROUND

If you change the Media type to **text/plain**  from   **application/json**   the correct formatting comes back **(see attached files of screen shots before and after)**

That is, in the example section,  we changed 
```
content:
    application/json
```
To
```
content:
    text/plain
```

And the formatting came back.

I assume this workaround isnt the solution, as we ARE dealing with JSON objects yes, so shouldn't the media type be  **application/json**

please advise

![screen shot of problem](https://user-images.githubusercontent.com/76476377/102883977-f1622b00-44a4-11eb-8871-6041af6183fa.png)
![screen shot with work around media type txt](https://user-images.githubusercontent.com/76476377/102883988-f58e4880-44a4-11eb-8051-8f62cf768435.png)

**Comment by hkosova** [comment]
The workaround is to remove `|-` at the beginning of JSON example values so that they are parsed as JSON objects instead of text.
```yaml
# Before
#              value: |-
#                { ...

# After
                value:
                  {
                      "in": "MSP.VSAM.DATA(REPRO1)"
                  }
```

**Comment by frankmainframecloud** [comment]
Brilliant, thanks!

 

 

Frank Fera | CTO

c/-Stone & Chalk

Level 4/11 York St, Sydney NSW 2000

M: +61 413 884 233

E:  <mailto:frank@mainframecloud.com> frank@mainframecloud.com

W:  <http://www.mainframecloud.com/> www.mainframecloud.com

 



 

CONFIDENTIALITY NOTICE: This email and any files transmitted with it are confidential and remain the property of Mainframe Cloud. Any disclosure, copying or further distribution is prohibited unless explicitly approved in writing by Mainframe Cloud.

**Comment by frankmainframecloud** [comment]
confirmed this resolved the problem.

while this used to work ok as is, and no doubt was caused by a swaggerhub change tightening its parsing.
It was a good practice to have been instructed on the correct syntax.

despite the REQUEST examples in our swagger definition still being ok in, we've applied the same change there to, so that they dont break when swaggerhub eventually tightens its parsing in that area too.

closing issue.
